test/main.test.ts

```diff
@@ -335,6 +335,78 @@ describe("Manual Action control", () => {
 
 });
 
+describe("Comments", () => {
+
+  const expectedLog = "Comments enabled but not yet supported";
+
+  test("does not comment if comments are disabled", async () => {
```

Comment:
Change wording for consistency with later `test`s in this block.

```suggestion
  test("don't comment if comments are disabled", async () => {
```